movingpandas/trajectory.py

```diff
@@ -691,10 +693,13 @@ def add_speed(self, overwrite=False):
         ----------
         overwrite : bool
             Whether to overwrite existing speed values (default: False)
+        name : str
+            Name of the speed column (default: "speed")
         """
-        if SPEED_COL_NAME in self.df.columns and not overwrite:
-            raise RuntimeError('Trajectory already has speed values! Use overwrite=True to overwrite exiting values.')
-        self.df = self._get_df_with_speed()
+        self.speed_col_name = name
+        if self.speed_col_name in self.df.columns and not overwrite:
+            raise RuntimeError(f'Trajectory already has a column named {self.speed_col_name}! Use overwrite=True to overwrite exiting values.')
```

Comment:
We could point out to the user that, alternatively, they can specify a different column name.